src/calendar/calendar.style.tsx

```diff
@@ -0,0 +1,124 @@
+import { Color } from "../color";
+import { ChevronLeftIcon } from "@lifesg/react-icons/chevron-left";
+import styled, { css } from "styled-components";
+import { TextStyleHelper } from "../text/helper";
+import { View } from "./types";
+
+interface HeaderDropdownProps {
+    $view: View;
+}
+
+interface CalendarViewProps {
+    children: JSX.Element[];
+    $show: View;
+}
+
+// =============================================================================
+// STYLING
+// =============================================================================
+export const Container = styled.div`
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    width: 41rem;
+    background-color: ${Color.Neutral[8]};
+    border: 1px solid ${Color.Neutral[5]};
+    height: 25.125rem;
+    border-radius: 0.75rem;
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+`;
+
+export const HeaderDropdown = styled.div<HeaderDropdownProps>`
+    display: flex;
+    justify-content: center;
+    margin-bottom: 1rem;
+
+    svg {
+        margin-left: 0.625rem;
+        color: ${Color.Neutral[3]};
+        transition: transform 250ms ease-in-out;
+    }
+
+    ${(props) => {
+        switch (props.$view) {
+            case "Month":
+            case "Year":
+                return css`
+                    & > :first-child {
+                        display: none;
+                    }
+                    & > :last-child svg {
+                        transform: rotate(180deg);
+                    }
+                `;
+        }
+    }}
+`;
+
+const DropdownMonthYearBase = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+
+    p {
+        ${TextStyleHelper.getTextStyle("H4", "regular")}
+    }
```

Comment:
Can extract this to a standalone `DropdownText` and replace `Text.BodySmall` in `calendar.tsx`